.github/workflows/macos-tests.yml

```
@@ -12,7 +12,7 @@ jobs:
   macos:
     name: Mac Py${{ matrix.PYTHON_VERSION }} - ${{ matrix.QT_BINDING }}
     timeout-minutes: 15
-    runs-on: macos-latest
+    runs-on: macos-12
```

```suggestion
    runs-on: macos-13
```

macOS 12 is going to removed from Github CIs in a few weeks.